client/src/plus/offer-overview/offer-overview-feature/index.tsx

```diff
@@ -28,6 +28,7 @@ export default function OfferOverviewFeatures() {
         img="/assets/notifications_light.png"
         imgAlt=""
       >
+        <h1>Notifications</h1>
         <h2>
           Development in real time:
           <br />
```

Comment:
We should not have multiple `h1` elements on a page. The `h1` should be reserved for the main page title. I would recommend that we change these to be `h2` and `h3` elements. An additional improvement would be to make each of these a section with an accessible name, that way there are seen as regions. Something like:

```html
<section aria-labelledby="notifications-section-title">
  <h2 id="notifications-section-title">Notifications</h2>
  <h3>...</h3>
</section>
```

Reply:
Good points!